README.md

```diff
@@ -21,6 +21,7 @@ To use the latest features of `torchtitan`, we recommend using the most recent P
 
 
 ## Latest News
+- [2025/10] SkyPilot now supports TorchTitan! See the tutorial [here](https://docs.skypilot.co/en/latest/examples/training/torchtitan.html).
```

Comment:
```suggestion
- [2025/10] SkyPilot now supports TorchTitan! Run distirbuted training with torchtitan on any clouds/Kubernetes: [tutorial](https://docs.skypilot.co/en/latest/examples/training/torchtitan.html).
```
Would something like this work? : )

Reply:
Oh sorry I merged it :( If everyone think this is better I could have a quick fix